### PR TITLE
Add Quad MxFE board support

### PR DESCRIPTION
**Pull request by dbogdan**

### CODE REVIEW

File: projects/ad9081/src/app_clock.c [hunk 1]
```diff
@@ -90,7 +77,7 @@ int32_t app_clock_init(struct clk clk[NUM_APP_CLKS])
 		.extra = &xil_spi_param
 	};
 
-	struct hmc7044_chan_spec chan_spec[8] = {
+	struct hmc7044_chan_spec chan_spec[6] = {
```
[the remark this answers is not in the document]
Comment (dbogdan) [reply projects/ad9081/src/app_clock.c]: Good remark - I've updated the code.

**Comment by dbogdan** [comment]
v2:
 - Automatically compute hmc7044_init_param num_channels.

File: projects/ad9081/src/app_parameters.h [hunk 2]
```diff
@@ -47,6 +47,8 @@
 #define PHY_CS	0
 #define CLK_CS	1
 
+#define PHY_RESET	55
```
[the remark this answers is not in the document]
Comment (mirceacaprioru) [reply projects/ad9081/src/app_parameters.h:63]: for consistency with PHY_CS i would keep like this

File: projects/ad9081/src/app_clock.c [hunk 3]
```diff
@@ -128,7 +115,7 @@ int32_t app_clock_init(struct clk clk[NUM_APP_CLKS])
 		.in_buf_mode = {0x07, 0x07, 0x00, 0x00, 0x15},
 		.gpi_ctrl = {0x00, 0x00, 0x00, 0x00},
 		.gpo_ctrl = {0x37, 0x33, 0x00, 0x00},
-		.num_channels = NUM_APP_CLKS,
+		.num_channels = 6,
```
Comment (buha) [inline projects/ad9081/src/app_clock.c]: make sure to use the channel number define here as well

**Comment by dbogdan** [comment]
v3:
 - Rebase on top of master.

File: projects/ad9081/src/app_clock.c (hunk 3 above)
Comment (dbogdan) [reply projects/ad9081/src/app_clock.c]: This is outdated code, isn't it?

Comment (buha) [reply projects/ad9081/src/app_clock.c]: yeah, sorry, i must've done something wrong that i saw and commented outdated code

**Pull request merged**